index.html

```diff
@@ -6,47 +6,37 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="./style.css">
-  <title>Document</title>
+  <title>Awesome books</title>
 
 </head>
 
 <body>
-  <header>
-    <h1>Awesome books</h1>
-  </header>
+
   <main>
     <section>
-      <ul  id="books">
-        <li>
-          <h3>Lorem Ipsum</h3>
-          <p>Testeroo Testy</p>
-          <button type="button">Remove</button>
-          <hr>
-        </li>
-        <li>
-          <h3>Second book</h3>
-          <p>Testeroo Testy</p>
-          <button class="deleteBook" type="button">Remove</button>
-          <hr>
-        </li>
+      <header>
+        <h1 class="awesome_books">Awesome books</h1>
+      </header>
+      <ul id="books">
+
       </ul>
       <script type="module">
         UI.displayBooks()
-      </script>      
+      </script>
```

Comment:
- [ ]  Please keep `script` tags before the closing `</body>` tag so the downloading and executing it doesn't block the browser to parse the `HTML` in the document. It also separates the `HTML` and `JavaScript` code and helps to manage the codebase better.

Reply:
Thank you for that keen observation. We have managed to place it right